gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/UserTaskController.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.gateway.rest;
+
+import static org.agrona.LangUtil.rethrowUnchecked;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import io.camunda.zeebe.auth.api.JwtAuthorizationBuilder;
+import io.camunda.zeebe.auth.impl.Authorization;
+import io.camunda.zeebe.broker.client.api.BrokerClient;
+import io.camunda.zeebe.broker.client.api.dto.BrokerRequest;
+import io.camunda.zeebe.broker.client.api.dto.BrokerResponse;
+import io.camunda.zeebe.gateway.protocol.rest.UserTaskCompletionRequest;
+import io.camunda.zeebe.gateway.rest.impl.broker.request.BrokerUserTaskCompletionRequest;
+import io.camunda.zeebe.msgpack.value.DocumentValue;
+import io.camunda.zeebe.protocol.impl.encoding.MsgPackConverter;
+import io.camunda.zeebe.protocol.record.value.TenantOwned;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import org.agrona.DirectBuffer;
+import org.agrona.concurrent.UnsafeBuffer;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@ZeebeRestController
+public class UserTaskController {
+
+  private final BrokerClient brokerClient;
+
+  @Autowired
+  public UserTaskController(final BrokerClient brokerClient) {
+    this.brokerClient = brokerClient;
+  }
+
+  @PostMapping(path = "/user-task/{userTaskKey}/complete", produces = "application/json")
+  public ResponseEntity completeUserTask(
+      final long userTaskKey, final UserTaskCompletionRequest request) {
+
+    return sendRequest(
+        request,
+        r -> {
+          return new BrokerUserTaskCompletionRequest(
+              userTaskKey, ensureJsonSet(request.getVariables()), request.getAction());
+        },
+        userTaskRecord -> ResponseEntity.status(HttpStatus.OK).build());
```

Comment:
@tmetzke we discussed with @npepinpe to avoid generalizing the `sendRequest` code for now.

Let's first add it directly to each controller, and see what can be generalized once we have usage in multiple places.

Regarding the error handling, Spring provides the `@ControllerAdvice` annotation which is used for error handling. Some docs and tutorials can be found here:
* https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/bind/annotation/ControllerAdvice.html
* https://www.baeldung.com/exception-handling-for-rest-with-spring